README.md

```diff
@@ -159,6 +159,12 @@ If you attempt to deploy a contract but you are not an `Admin` not
 a `Deployer`, you will see something like:
 ![deploy fail](./imgs/deploy_fail.png)
 
+The allow list has three roles: `None`, `Deployer`, and `Admin`.
+
+If you call `readAllowList(addr)` then you can read the current role of `addr`, which will return a uint256 with a value of 0, 1, or 2, corresponding to the roles `None`, `Deployer`, and `Admin` respectively.
+
+WARNING: if you remove all of the admins from the allow list, it will no longer be possible to update the allow list without modifying modifying the subnet-evm to schedule a network upgrade.
```

Comment:
```suggestion
WARNING: if you remove all of the admins from the allow list, it will no longer be possible to update the allow list without modifying the subnet-evm to schedule a network upgrade.
```